src/components/Modals/ProfileSaveReminder.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <teleport-modal
+    title="Warning: Possibly Unsaved Changes"
+    leftButtonText="Go Back"
+    rightButtonText="Continue"
+    content-class="content-clear"
+    @modal-closed="returnToProfilePage"
+    @left-button-clicked="returnToProfilePage"
+    @right-button-clicked="confirmLeaveProfilePage"
+  >
+    <span
+      >There may be unsaved changes in your profile! Please make sure your changes are saved before
+      leaving!</span
+    >
```

Comment:
Can we clean up this formatting a bit?

Reply:
this was the formatting suggested by VSCode